isis/src/base/objs/CSMSkyMap/CSMSkyMap.cpp

```diff
@@ -0,0 +1,45 @@
+/** This is free and unencumbered software released into the public domain.
```

Comment:
Do we want to merge #5069 before this PR so that we can get a clean file history?  Not sure how we want to handle the similar PRs.

Reply:
I think we can merge this in favor of #5069, this PR should be a super set of that PR